cpp/cmake/Modules/JitifyPreprocessKernels.cmake

```diff
@@ -38,7 +39,7 @@ function(jit_preprocess_files)
       COMMAND
         jitify_preprocess ${ARG_FILE} -o
         ${CUDF_GENERATED_INCLUDE_DIR}/include/jit_preprocessed_files -i -m -std=c++17
-        -remove-unused-globals -D__CUDACC_RTC__ -I${CUDF_SOURCE_DIR}/include
+        -remove-unused-globals -D_FILE_OFFSET_BITS=64 -D__CUDACC_RTC__ -I${CUDF_SOURCE_DIR}/include
```

Comment:
We should replicate the docs around why we pass _FILE_OFFSET_BITS here.

Reply:
Done -- thanks!